Formula/llvm.rb

```diff
@@ -254,11 +267,8 @@ def install
         cxxflags << "-isystem#{toolchain_path}/usr/include"
         cxxflags << "-isystem#{macos_sdk}/usr/include"
       elsif !OS.mac?
-        gcc = Formula["gcc"]
-        # Link to libstdc++ for brewed GCC rather than the host GCC which is too old.
-        # Also make sure brewed glibc will be used if it is installed.
+        # Make sure brewed glibc will be used if it is installed.
         linux_library_paths = [
-          gcc.opt_lib/"gcc"/gcc.version.major, # libstdc++
```

Comment:
We have to replace this with `HOMEBREW_PREFIX/lib` or else the linker won't be able to find `libstdc++` on systems that install `gcc@11` because their host GCC is too old.

Reply:
Ok, next push. Currently this code is not reached.